fi.laamaa.m8c.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=M8C
+GenericName=Dirtywave M8 Client
+Exec=/usr/bin/m8c
```

Comment:
this is incorrect.

1. it should be just the executable name (not absolute path)
2. it's definitely not in `/usr`.